.github/workflows/test-rust.yaml

```diff
@@ -79,7 +76,9 @@ jobs:
           runner.os == 'Linux') }}
         with:
           command: insta
-          args: test --unreferenced=auto ${{ inputs.target_option }}
+          args:
+            test --unreferenced=auto ${{ inputs.target_option }} --features
+            test-external-dbs
```

Comment:
Here's what I meant in the other comment — using the `os` value to determine whether to include the `--features` option. Does that make sense?

```suggestion
            test --unreferenced=auto ${{ inputs.target_option }}
            ${{ inputs.os == 'ubuntu-latest' && '--features=test-external-dbs' || '' }}
```